bibliopixel/animation/animation.py

```diff
@@ -43,8 +43,8 @@ def completed(self, state):
         else:
             self.state = STATE.running
 
-    def preRun(self, amt=1):
-        self.layout.all_off()
+    def preRun(self):
```

Comment:
Haha... damnit. I should've done that. Yeah... that's technically a breaking change. I guess I could do a `self.preRun = self.pre_run` thing somewhere but I'm not sure that will work when classes are overriding this method. Ehh... I'm fine with just changing it and I can update all of BPA at the same time.